bundles/AdminBundle/Controller/Admin/Document/DocumentController.php

```diff
@@ -48,6 +48,7 @@
 class DocumentController extends ElementControllerBase implements EventedControllerInterface
 {
     use AdminStyleTrait;
```

Comment:
Is the AdminStyleTrait still needed here ?

Reply:
yes, still calling here: https://github.com/pimcore/pimcore/blob/969f4efbc7c9a480690e366dc0d0e1a9b5940440/bundles/AdminBundle/Controller/Admin/Document/DocumentController.php#L106

https://github.com/pimcore/pimcore/blob/969f4efbc7c9a480690e366dc0d0e1a9b5940440/bundles/AdminBundle/Controller/Admin/Document/DocumentController.php#L785

Reply:
oh, you are right. we don't need it, since it is already used in DocumentTreeConfigTrait